src/components/DatePicker/DatePicker.Props.ts

```diff
@@ -67,6 +67,11 @@ export interface IDatePickerProps extends React.Props<DatePicker> {
    * Localized strings to use in the DatePicker
    */
   strings: IDatePickerStrings;
+
+  /**
+  * Locales to use for rendering date strings, in descending order of priority
```

Comment:
Could you provide an example here? is this "en-us" or "1033"? Is there a list somewhere of valid locales that is supported?